templates/quickstart-database-for-atlassian-services.yaml

```diff
@@ -12,6 +12,14 @@ Parameters:
       - 'PostgreSQL'
     ConstraintDescription: Must be 'Amazon Aurora PostgreSQL' or 'PostgreSQL'.
     Type: String
+  DBPostgresEngineVersion:
```

Comment:
Can we use this with the Aurora DB as well?

https://github.com/aws-quickstart/quickstart-amazon-aurora-postgresql/blob/develop/templates/aurora_postgres.template.yaml#L162

Reply:
That was something I wanted to discuss. See this PR as a driver for these changes, I'm going to schedule a call on this